Makefile

```diff
@@ -322,6 +322,16 @@ azure-secrets:
 	oc create secret generic codecov-token --from-literal=upload=${CODECOV_UPLOAD_TOKEN} -o yaml --dry-run | oc apply -n azure -f -
 .PHONY: azure-secrets
 
+vsphere-secrets:
+	oc create secret generic cluster-secrets-vsphere \
+	--from-file=cluster/test-deploy/vsphere/.awscred \
+	--from-file=cluster/test-deploy/vsphere/ssh-privatekey \
+	--from-file=cluster/test-deploy/vsphere/ssh-publickey \
+	--from-file=cluster/test-deploy/vsphere/pull-secret \
+	--from-file=cluster/test-deploy/vsphere/secret.auto.tfvars \
+	-o yaml --dry-run | oc apply -n ocp -f -
+.PHONY: vsphere-secrets
+
```

Comment:
This is not necessary for secrets already in Bitwarden.

Reply:
Thanks, dropped this part